packages/core/src/dataTypes/integer/index.ts

```diff
@@ -26,3 +26,32 @@ export function integer(
     },
   }
 }
+
+// type Prettify<T> = {
```

Comment:
i'll remove this before merging!

Reply:
but if you pull this down and uncomment the code here, you can see the issue that is being solved